CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+litomore@gmail.com.
```

Comment:
Which email should I use here?

Reply:
@mondeja - do we have capacity for a `community@simpleicons.org` email? There's the potential there for us to split the 'teams' working on the project to those dealing with legal enquiries through the `removals@` address, and those tackling more community-facing activity through `community@`.

Reply:
We are currently using Zoho for the `removals@simpleicons.org` email and I'm not even an admin of the Zoho account/organization. Pinging @service-paradis as he is the superadmin of the account.

Reply:
Just created `community@simpleicons.org`. It can be used!

Right now, email members are Eric Cornelissen and @sachinraja.
We might replace Eric with someone else. @adamrusted was as a member in the past. Do you want to be a member again?

Reply:
More than happy to, yeah. If you want to replace Eric's with mine (assuming you can re-activate/create mine) and have the community@ hit mine by default?

Reply:
Just created your account. `community@` and `removals@` are distribution list where emails will be sent in every members inbox.